src/pokemon-client.js

```diff
@@ -0,0 +1,36 @@
+class PokemonClient {
+   constructor() {
+      this.API_URL = "https://pokeapi.co/api/v2/pokemon/";
+   }
+
+   async getPokemon(pokemonID) {
+      try {
+         const response = await fetch(this.API_URL + pokemonID);
+         const data = await response.json();
+
+         return data;
+      } catch (error) {
+         return false;
+      }
+   }
+
+   async getAllPokemonNames() {
+      try {
+         const numberOfPokemons = 898;
+         const response = await fetch(
+            this.API_URL + "?limit=" + numberOfPokemons
```

Comment:
You can use `URLSearchParams` to make it more readbale
You have only 1 param here so it's not a big deal. up to you
https://stackoverflow.com/a/49701878/3762628

Reply:
Cool. I  just got rid of this number so no need anymore.